README.md

```diff
@@ -89,7 +104,37 @@ samply record ./target/profiling/egglog tests/extract-vec-bench.egg
 env RUST_LOG=error samply record ./target/profiling/egglog --dont-print-messages tests/extract-vec-bench.egg
 ```
 
-# Documentation
-
-To view documentation, run `cargo doc --open`.
+**Pay Attention to Parallelism** egglog has support to run programs in parallel
```

Comment:
@ezrosent By the way, how many threads are used if we call egglog's rust `parse_and_run_program` method? Is it also 1, or all threads?

Reply:
This seems like a weird behavior...